ember-cli-build.js

```diff
@@ -137,8 +137,11 @@ module.exports = function (defaults) {
     app.import('bower_components/google-caja/html-css-sanitizer-bundle.js');
     app.import('bower_components/jqueryui-touch-punch/jquery.ui.touch-punch.js');
 
+
+
     if (app.env === 'test') {
         app.import(app.bowerDirectory + '/jquery.simulate.drag-sortable/jquery.simulate.drag-sortable.js', {type: 'test'});
+        app.import('ghost-admin/transforms/json-string.js');
```

Comment:
This import shouldn't be needed as Ember's resolver should pick it up automatically, if it was causing issues in testing perhaps it was missing from the `needs` array in unit tests?